forge/db/views/Project.js

```diff
@@ -210,6 +210,39 @@ module.exports = function (app) {
         return result
     }
 
+    app.addSchema({
+        $id: 'DashboardInstanceSummary',
+        type: 'object',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            url: { type: 'string' },
+            createdAt: { type: 'string' },
+            updatedAt: { type: 'string' },
+            links: { $ref: 'LinksMeta' },
+            application: { type: 'object', additionalProperties: true },
```

Comment:
```suggestion
            application: { $ref: 'ApplicationSummary' },
```

We should be explicit about the schema

Reply:
Nice catch!